src/OpenTelemetry.Abstractions/Metrics/IMetric{T}.cs

```diff
@@ -0,0 +1,72 @@
+﻿// <copyright file="IMetric{T}.cs" company="OpenTelemetry Authors">
```

Comment:
is it the new way of naming files with generics?

Reply:
yep. Stylecop asks for it